<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myicon.qrc">
    <normaloff>:/myimages/beijing.jpg</normaloff>:/myimages/beijing.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background-image: url(:/myimages/change.jpg);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>小小工具箱</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="DecisionButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color:rgb(148,203,172);
	border-radius:5px;
	color: rgb(255, 255, 255);
}
QPushButton:hover
{
	background-color:rgb(122,189,154);
	border-radius:5px;
}
</string>
    </property>
    <property name="text">
     <string>做个决定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DateButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color:rgb(148,203,172);
	border-radius:5px;
	color: rgb(255, 255, 255);
}
QPushButton:hover
{
	background-color:rgb(122,189,154);
	border-radius:5px;
}
</string>
    </property>
    <property name="text">
     <string>纪念日</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculatorButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color:rgb(148,203,172);
	border-radius:5px;
	color: rgb(255, 255, 255);
}
QPushButton:hover
{
	background-color:rgb(122,189,154);
	border-radius:5px;
}
</string>
    </property>
    <property name="text">
     <string>亲属关系计算</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="myicon.qrc"/>
 </resources>
 <connections/>
</ui>
